### PR TITLE
Add lograge

### PR DESCRIPTION
**Pull request by gravesm**
This configures lograge to handle logging by default. It can be disabled
by setting the DISABLE_LOGRAGE env var.

### CODE REVIEW

File: config/initializers/lograge.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Rails.application.configure do
+  unless ENV["DISABLE_LOGRAGE"].present?
+    config.lograge.enabled = true
+  end
+
+  config.lograge.custom_options = lambda do |event|
+    exceptions = %w(controller action format id)
+    {
+      params: event.payload[:params].except(*exceptions),
+      time: event.time.utc().iso8601(3),
```
Comment (JPrevost) [inline config/initializers/lograge.rb:10]: Cool. Do you think this time format is what we'll standardize on for centralized logging?

File: config/initializers/lograge.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    }
+  end
+
+  # This is mostly only useful when central logging is set up
```
Comment (JPrevost) [inline config/initializers/lograge.rb:14]: 👍

**Review by JPrevost — APPROVED** [review]
🌈

**Comment by JPrevost** [comment]
We're gonna need to battle it out on opt-in vs opt-out lol.

In general I think this approach is solid (aka as an initializer instead of per env). 👍

File: config/initializers/lograge.rb (hunk 1 above)
Comment (gravesm) [reply config/initializers/lograge.rb:10]: I don't really have a strong argument for this other than it's what I prefer.

**Pull request merged**